tests/test_pdf.py

```diff
@@ -1,8 +1,29 @@
+import re
+from pathlib import Path
+from typing import List
+
 import pytest
 
 
 @pytest.mark.sphinx("latex", testroot="simple")
-def test_simple(tex: str):
-    assert r"\sphinxincludegraphics[]" \
-           r"{drawio-6cf867c6e94665d8489581a35c2d215220d6c152.pdf}" in tex
+def test_simple(tex_images: List[Path]):
+    image, = tex_images
+    assert image.basename() == "drawio-6cf867c6e94665d8489581a35c2d215220d6c152.pdf"
+
+
+@pytest.mark.sphinx("latex", testroot="simple")
+def test_crop(tex_images: List[Path]):
+    image, = tex_images
+    assert get_mediabox(image) == ("88.080002", "46.079998")
+
 
+def get_mediabox(filename):
+    with filename.open("rb") as pdf:
+        for line in pdf.readlines():
+            if line.startswith(b"/MediaBox"):
+                break
+        else:
+            assert False
+    m = re.match(rb"/MediaBox \[0 0 ([\d.]+) ([\d.]+)\]", line)
+    assert m
```

Comment:
I'd prefer if this raised a more informative exception rather than just an AssertionError (or at least, have a message).

Reply:
This is a test, so I think we need to assert? Added a message now.

Reply:
This isn't directly a test - it's a function called by it.
Not sure how _exactly_ that works with pytest.
If you could do similar for `assert False` (which seems like it should just be something like `raise Exception` to me), that'd be great.

Once that's done I'll merge this and # 33

Reply:
> This isn't directly a test - it's a function called by it.

True. The assert does seem to do the job, so let's use that.

I refactored the function to have just a single assert statement.

I'll rebase #33 when you merge this one.